### ISSUE TITLE
Outbound Communication Overview

### ISSUE THREAD

**Issue by Merelleya** [HaxeFoundation/Project-Management#5]
### Topics
- Development News
  - Roadmap
  - Features
  - What is being worked on
  - Status updates
- Haxe Foundation Updates 
  - Project Updates
  - Financial Reports
  - Future Projects
  - Calls for Help
  - Events
  - What do we do?
  - How do we do it?

We may also include adequate responses to escalating issues to make our position more clear. In addition, we will want to address the following topics:
- Who is reponsible for what?
- Where do I get help?
- How can I contribute?
- Best practices for contributions, discussions, etc.

And then there is "marketing" or feature content about:
- the eco-system
- How Haxe is being used professionally
- Special things one can do with Haxe
### Channels
- Haxe.org Website
- Haxe Blog
- Showcases
- Press Releases
- Social Media
- Plain old E-Mail
### Content Types
- Blog Posts and Articles
  - For insight into processes
  - Development News
  - Transparency Posts
  - Features and showcases
  - As a Wrap Up after conferences and meetups
  - To summarize meeting outcomes
  - To summarize PM-Discussions
- Paid press releases
  - For Releases
  - New Partners and Partner Renewals
  - Events we organize
  - Maybe for certain speaking engagements
- Social Media
  - Support our content
  - Curate Content from other members of the community
  - Curate content from other parts of the eco-system
  - Curate meta-content (for example about other open-source projects)
  - Specific Campaigns
  - Support Events

It should be clear in which cases we use what medium. The one where I have most concerns are paid press releases. I think we should have a Press Kit online and do Paid releases on an event basis.

**Comment by Merelleya** [comment]
I have started with a summary of the different channels and content types. I have as yet not included actual face-to-face events. I think that we will need to find different specific guidelines and campaigns for each of the channels.  

We might have to have a very meta-groundfinding-session about what our actual _**core**_ messages are (the product message, so to speak). They will then help come up with the general "tone" of communications and find our "voice".

**Comment by kulick** [comment]
I think this list looks good, but ambitious.  I think it might be helpful to choose a few areas in which to apply your initial focus.  I wouldn't be surprised to hear you say that you already plan something like that.  If so, could you name the areas of focus for, say, the next 3 months?  6 months?

I have taken an action item to reach out to my press team regarding a joint press release.  I will follow up with you once I have talked to them.

**Comment by Merelleya** [comment]
My number one priority at the moment is launching the showcases, keeping them coming and get the Blog rolling. Next, I will probably want to tackle a press section on the website and a press kit as well as re-arrange some neglected parts on the website to make it a little better structured.

I do have an eye on the social media daily and cross post some things, but at the moment I don't have enough content to support a full blown campaign (like...let's say a #HaxeIt campaign on twitter).

Press Releases is the one I am most uncomfortable with, mainly because I don't have any proper Data to evaluate whether the time spent is worth it or not. I have even looked at some of the clients where I work and they deal SO differently with this, so I guess it is really a case-by-case thing. I estimated that it would take about 10 to 20 hours in time to get a polished text (coordinated with everybody and edited and so on) plus the cost for the release itself. I have yet to decide whether that would be time/money well spent. Especially, if it would mean the other areas get less attention due to time constraints.

**Comment by ncannasse** [comment]
@Merelleya PR is long term investment : it will take a lot of time at first, and don't expect immediate profitability for it. But the next time it will take less time, and we will have more visibility, and even better next time, etc. Let's get started asap so we start building up our skills/reputation

**Comment by Merelleya** [comment]
All right then. I propose we start with something like TiVo being our partner for the third year or something product related. Input more than welcome.

**Comment by ncannasse** [comment]
Yes, something sounding like "TiVo and Haxe Foundation partnership strengthen , TiVo to generalize Haxe adoption for all its products" (which is what I understood from WWX slides) -- followed by success story description about increasing adoption of Haxe in TiVo and how it works good for them, quotes from myself and Todd? about how we are happy to work together etc (the usual PR stuff)

@kulick tell me if that would work for you and your PR dept

**Comment by kulick** [comment]
Need to run by my PR guy.  I'll try to catch him tmw and get back to you...

**Comment by MatthijsKamstra** [comment]
Curious where we are on the subject!

**Comment by Merelleya** [comment]
revisiting this again. Will try to sort out the necessary infrastructure for Blogs/Articles with @jasononeil in the coming days.

**Comment by Merelleya** [comment]
Closing this for now. The blog is somewhere almost ready and for the rest I will open separate issues if necessary.
